readme.md

```diff
@@ -82,20 +84,35 @@ require('@zeit/ncc')('/path/to/input', {
   v8cache: false, // default
   quiet: false, // default
   debugLog = false // default
-}).then(({ code, map, assets }) => {
-  console.log(code);
-  // Assets is an object of asset file names to { source, permissions, symlinks }
-  // expected relative to the output code (if any)
+}).then(({ files, symlinks }) => {
```

Comment:
I think you need to change `/path/to/input` to `build.js` (I can't comment on line 72 for some reason)

Reply:
Great catch, it's actually `index.js` :)